Add toggle button in replacement of upper and lower

Vishal1297 opened the pull request.

Vishal1297 commented on src/components/TextForm.js:
```diff
@@ -24,14 +24,8 @@ export default function TextForm(props) {
-      }
-    });
-    setText(string);
+    let txtWithoutTrailingLeadingSpaces = text.trim().split(" ").join(" ").trim();
```
while appending one extra space will be left, so removed using trim

Vishal1297 replied:
@Suyash99 

Fixed

Vishal1297 commented on src/components/TextForm.js:
```diff
@@ -44,6 +38,17 @@ export default function TextForm(props) {
     setText(lowerCaseText);
   };
 
+  /**
```
Will be used for single button for case toggling.

Vishal1297 replied:
@Suyash99 

fixed

Suyash99 commented:
@Vishal1297 Please push these changes on new Branch. Will pull and test these changes

Vishal1297 commented:
> @Vishal1297 Please push these changes on new Branch. Will pull and test these changes

Why new branch ?

Vishal1297 commented:
## Screenshot

![image](https://user-images.githubusercontent.com/43027336/194745561-d9ae6211-2d64-409c-8922-8767ce59731c.png)


### Done

- [x] Added Toggle
- [x] Fixed remove spaces
